nixos/modules/config/xdg/icons.nix

```
@@ -13,6 +13,14 @@
         [XDG Icon Theme specification](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html).
       '';
     };
+    xdg.icons.fallbackThemes = lib.mkOption {
```

Maybe this to avoid confusion?
```suggestion
    xdg.icons.fallbackCursorThemes = lib.mkOption {
```